<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>precastingWidgetClass</class>
 <widget class="QWidget" name="precastingWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>51</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>51</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>precastingWidget</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="hint_dist_tr0">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 16pt;
color: white</string>
       </property>
       <property name="text">
        <string>距离</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hint_subjName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 18pt;
color: white</string>
       </property>
       <property name="text">
        <string>语文下课</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hint_dist_tr1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>5</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 16pt;
color: white</string>
       </property>
       <property name="text">
        <string>还有</string>
       </property>
       <property name="textFormat">
        <enum>Qt::TextFormat::RichText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hint_timeNum">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 39, 39);
font-size: 28pt;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="text">
        <string>56</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hint_unit">
       <property name="styleSheet">
        <string notr="true">font-size: 16pt;
color: white</string>
       </property>
       <property name="text">
        <string>秒</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
